### PR TITLE
[KD] Stylize Header and Navbar

### LINKED ISSUE #80: Stylize the Navbar
## Summary
We've decided on some brand styles for Aisle Be There:

- Font: Righteous
- Colors: purple, black, white

Now we want to stylize our navigation bar with these attributes while keeping the design accessible.

## Acceptance criteria
- [x] Stylize the navbar with the brand colors

## Notes
Sub-issue of https://github.com/the-collab-lab/tcl-69-smart-shopping-list/issues/14

### PR DESCRIPTION
**Pull request by piecanoe**
## Description

This code stylizes the header, navbar, and sign-in/sign-out button using the Aisle Be There brand colors and font.

## Related Issue

Closes #80
Sub-Issue #14

## Acceptance Criteria

- [x] Stylize the navbar with the brand colors

## Type of Changes

Use one or more labels to help your team understand the nature of the change(s) you’re proposing. E.g., `bug fix` or `enhancement` are common ones.

### Before

before_1

### After

Signed out
after_1

Signed in
after_2

## Testing Steps / QA Criteria
1. From 'Home' page, make you're signed out.
2. Click the "here" to learn more link to navigate to the 'About' page.
3. You should see a "Sign In" button at the top right. Hover over the button to see that it changes to a darker purple.
4. Click "Sign In" to sign in.
5. Once you're signed in, the "Sign In" button should change to a "Sign Out" button with your user profile image. Hover over the button to see that it changes to a darker purple.
6. While still signed in, click on between the "lists" and "about" nav links to see that the active page is underlined while the other nav links are not.
7. Click on "Aisle Be There" in the header to verify it navigates you to the home page.

### CODE REVIEW

**Review by krsnamara — APPROVED** [review]
Very exciting to see the app starting to take shape design wise.

File: src/views/Layout.css [hunk 1]
```diff
@@ -18,14 +18,58 @@
...
+
+.Layout-header-profile {
+	border-radius: 50%;
+	border: 1px solid transparent;
+	width: 40px;
+}
+
+.Layout-header button {
+	background: transparent;
+	color: var(--color-text);
```
Comment (krsnamara) [inline src/views/Layout.css]: Looking at the deployed it appears that this overrides the button styles in index making the text in the signin/signout button black. Suggestion is to remove this line

Comment (krsnamara) [inline src/views/Layout.css]:
```suggestion
```

**Review by krsnamara — COMMENTED** [review]
Added a comment and suggestion to delete text

File: src/views/Layout.css (hunk 1 above)
Comment (piecanoe) [reply src/views/Layout.css]: Thanks for catching that! I will also change the 'button' element text color in 'index.css' to white so the buttons are readable in dark and light mode.

Comment (piecanoe) [reply src/views/Layout.css]: Thanks for catching that. I'll remove that line and also change the text color of 'button' in index.css to '--color-white' so it applies to all  buttons in both light and dark mode.

**Review by amalyam — APPROVED** [review]
Good work, looks really nice! One suggestion to bring app name and navbar items a little closer to the center, things feel a bit close to the edge and navbar items feel a bit far apart.

Also, sign out button looks right for me in chrome, but looks like this in fIrefox:
Screen Shot 2024-04-05 at 22 56 59

I'm not sure what's causing the difference between the two browsers, would have to do more digging. Maybe @mxmason has thoughts?

**Comment by amalyam** [comment]
update to comment above^: after setting the firefox window width to the narrowest it can be, the signout button looks fine, so I guess it's how the button is scaling up at full view? I also see now how setting the location of the app name and navbar items closer to the center might be tricky to have it both look good at the narrowest screen size and full view without getting into media queries, so probably doesn't make sense for now unless you wanted to play around with that if you have time.

**Pull request merged**